src/dodal/devices/i20/gas_injector.py

```diff
@@ -0,0 +1,201 @@
+import enum
+import time
+from collections import deque
+
+from ophyd_async.core import StandardReadable, WatcherUpdate, observe_value
+from ophyd_async.epics.core import epics_signal_r, epics_signal_rw
+
+
+class GasToInject(enum.Enum):
+    ARGON = "argon"
+    HELIUM = "helium"
+    KRYPTON = "krypton"
+    NITROGEN = "nitrogen"
+
+
+class IonChamberToFill(enum.Enum):
+    i0 = "I0"
+    i1 = "I1"
+    iT = "iT"
+    iRef = "iRef"
```

Comment:
~~These should extend ophyd_async's StrictEnum~~ 
only if they're being used as Signal types. But they're not